packages/core/tests/unit/models/pie-chart-test.ts

```diff
@@ -0,0 +1,562 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import { buildTestRequest, Column } from '../../helpers/request';
+
+module('Unit | Model | Pie Chart Visualization Fragment', function(hooks) {
+  setupTest(hooks);
+
+  test('default value', function(assert) {
+    assert.expect(1);
+
+    const metricsAndDims: Column[][] = [
+      [
+        { field: 'm1', cid: '1' },
+        { field: 'm2', cid: '2' }
+      ],
+      [
+        { field: 'd1', cid: '3' },
+        { field: 'd2', cid: '4' }
+      ]
+    ];
+    const chart = this.owner
+      .lookup('service:store')
+      .createRecord('all-the-fragments')
+      .get('pieChart');
```

Comment:
Can this be in a beforeEach? Also make sure to update the definition in `all-the-fragments`
https://github.com/yahoo/navi/blob/a95c3045f8f8eba7153a33f5df8aa7e0416f8bbf/packages/core/tests/dummy/app/models/all-the-fragments.ts#L19-L20
```ts
const Store = this.owner.lookup('service:store') as StoreService;
const { pieChart } = Store.createRecord('all-the-fragments');
```